test/Rentals.spec.ts

```diff
@@ -3424,6 +3579,51 @@ describe('Rentals', () => {
       await estate.connect(lessor)['safeTransferFrom(address,address,uint256,bytes)'](lessor.address, rentals.address, estateId, bytes)
     })
 
+    it('should allow a lessor to accept the offer of a smart contract account. The smart contract account can then act as a tenant', async () => {
```

Comment:
What do you think of adding the next time to differentiate tests with the same description to add something like this:

```suggestion
    it('should allow a lessor to accept the offer of a smart contract account. The smart contract account can then act as a tenant' :: safeTransferFrom, async () => {
```

Reply:
I think that differentiation is made by the "describe" that test is inside. different would be to have 2 different tests with the same name in the same place.

Reply:
But I understand your point when trying to search for the test in the test file. 
Having 2 different matches can be confusing.